src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -271,6 +272,7 @@
     <ProjectReference Condition="'$(TargetGroup)'=='netfx' AND $(ReferenceType)=='Project'" Include="$(NetFxSource)src\Microsoft.Data.SqlClient.csproj" />
     <ProjectReference Include="$(AddOnsPath)AzureKeyVaultProvider\Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.csproj" />
     <ProjectReference Condition="$(ReferenceType.Contains('NetStandard'))" Include="$(TestsPath)NSLibrary\Microsoft.Data.SqlClient.NSLibrary.csproj" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.12.1" />
```

Comment:
@DavoudEshtehari do i need to do some special here for the version?

Reply:
If you are going to add a 3rd party to the test project you can define its version in the [Test Project Dependencies](https://github.com/cmeyertons/SqlClient/blob/master/tools/props/Versions.props#L52) section at [Versions.props](https://github.com/cmeyertons/SqlClient/blob/master/tools/props/Versions.props).